src/sentry_plugins/__init__.py

```diff
@@ -3,5 +3,23 @@
 try:
     VERSION = __import__('pkg_resources') \
         .get_distribution('sentry-plugins').version
-except Exception, e:
+except Exception as e:
     VERSION = 'unknown'
+
+# Try to hook our webhook watcher into the rest of the watchers
+# iff this module is installed in editable mode.
```

Comment:
Why does nobody know what this means? 😡 

https://en.wikipedia.org/wiki/If_and_only_if